### PR TITLE
[zh-cn] sync blog 2023-07-20-sig-cli-spotlight.md

### PR DESCRIPTION
**Pull request by my-git9**
sync blog 2023-07-20-sig-cli-spotlight.md

### CODE REVIEW

File: content/zh-cn/blog/_posts/2023-07-20-sig-cli-spotlight.md [hunk 1]
```diff
@@ -0,0 +1,297 @@
+---
+layout: blog
+title: "聚焦 SIG CLI"
+date: 2023-07-20
+slug: sig-cli-spotlight-2023
+canonicalUrl: https://www.kubernetes.dev/blog/2023/07/13/sig-cli-spotlight-2023/
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2023-07-20-sig-cli-spotlight.md:6]: line 6 can be removed in zh version

File: content/zh-cn/blog/_posts/2023-07-20-sig-cli-spotlight.md [hunk 2]
```diff
@@ -0,0 +1,297 @@
...
+aligned with my experience from internal projects. I started regularly
+attending meetings and made a few small PRs, and by 2021 I was working
+more deeply with the
+[Kustomize](https://github.com/kubernetes-sigs/kustomize) team
+specifically. Later that year, I was appointed to my current roles as
+subproject owner for Kustomize and KRM Functions, and as SIG CLI Tech
+Lead and Chair.
+-->
+**Katrina**：自 2016 年以来，我一直作为最终用户使用 Kubernetes，但直到 2019 年底，
+我才发现 SIG CLI 与我在内部项目中的经验非常吻合。我开始定期参加会议并制作了一些小型 PR，
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2023-07-20-sig-cli-spotlight.md:95]:
```suggestion
我才发现 SIG CLI 与我在内部项目中的经验非常吻合。我开始定期参加会议并提交了一些小型 PR，
```

File: content/zh-cn/blog/_posts/2023-07-20-sig-cli-spotlight.md [hunk 3]
```diff
@@ -0,0 +1,297 @@
...
+
+<!--
+## Current projects and challenges
+
+**Arpit**: Going through the README file, it’s clear SIG CLI has a
+number of subprojects, could you highlight some important ones?
+-->
+## 当前的项目和挑战
+
+**Arpit**：查看 README 文件可以发现 SIG CLI 有许多子项目，你们能提一些重要的子项目吗？
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2023-07-20-sig-cli-spotlight.md:157]:
```suggestion
**Arpit**：阅读了一遍 README 文件，发现 SIG CLI 有许多子项目，你能突出讲一些重要的子项目吗？
```

File: content/zh-cn/blog/_posts/2023-07-20-sig-cli-spotlight.md [hunk 4]
```diff
@@ -0,0 +1,297 @@
...
+  template-free customization tool for Kubernetes yaml manifest files.
+* [KUI](https://kui.tools) - a GUI interface to Kubernetes, think
+   `kubectl` on steroids.
+* [`krew`](https://github.com/kubernetes-sigs/krew): a plugin manager for `kubectl`.
+-->
+**Maciej**：在我看来，值得你投入时间的四个最活跃的子项目是：
+
+* [`kubectl`](https://github.com/kubernetes/kubectl)：规范的 Kubernetes CLI。
+* [Kustomize](https://github.com/kubernetes-sigs/kustomize)：Kubernetes yaml 清单文件的无模板定制工具。
+* [KUI](https://kui.tools) - Kubernetes 的 GUI 界面，想想类固醇上的 `kubectl`。
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2023-07-20-sig-cli-spotlight.md:174]:
```suggestion
* [KUI](https://kui.tools) - 一个针对 Kubernetes 的 GUI 界面，可以将其视为增强版的 `kubectl`。
```

File: content/zh-cn/blog/_posts/2023-07-20-sig-cli-spotlight.md [hunk 5]
```diff
@@ -0,0 +1,297 @@
...
+users the ability to set or override default values we can’t change,
+and even create custom subcommands via aliases
+-->
+**Katrina**：我们每个决定面临的最大挑战是向后兼容性并确保我们不会影响现有用户。
+经常发生的情况是，修复表面上的内容似乎很简单，但即使修复 bug 也可能对某些用户造成破坏性更改，
+这意味着我们需要经历一个较长的弃用过程来更改它，或者在某些情况下我们不能完全改变它。
+另一个挑战是我们需要在工具上公开 flag 的平衡定制和可用性。例如，我们收到了许多关于新标志的建议，
+这些建议肯定对某些用户有用，但没有足够大的子集来证明，将它们添加到工具中对每个用户来说都会增加复杂性。
+`kuberc` 提案可能会帮助个人用户设置或覆盖我们无法更改的默认值，甚至通过别名创建自定义子命令，
+从而帮助解决其中一些问题
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2023-07-20-sig-cli-spotlight.md:238]:
```suggestion
从而帮助解决其中一些问题。
```
中文可以加 `。`

**Review by windsonsea — COMMENTED** [review]
/approve

**Review by ydFu — APPROVED** [review]
/lgtm

**Pull request merged**